ARM-wvd-templates/Update exisiting WVD host pool/README.md

```diff
@@ -0,0 +1,112 @@
+Click the button below to deploy:
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FRDS-Templates%2Fmaster%2FARM-wvd-templates%2FUpdate%20existing%20WVD%20host%20pool%2FmainTemplate.json" target="_blank">

```

Comment:
Need to change mainTemplate.json to AddVirtualMachinesTemplate.json

Reply:
ok